False positive: indentation inside of constructor parameters inside try block

I have read check documentation: https://checkstyle.sourceforge.io/checks/misc/indentation.html
I have downloaded the latest checkstyle from: https://checkstyle.org/cmdline.html#Download_and_Run
I have executed the cli and showed it below, as cli describes the problem better than 1,000 words

```cmd
E:\temp\CheckstyleReport>type config.xml
<?xml version="1.0"?>
<!DOCTYPE module PUBLIC
          "-//Puppy Crawl//DTD Check Configuration 1.3//EN"
          "https://checkstyle.org/dtds/configuration_1_3.dtd">
<module name="Checker">
  <property name="tabWidth" value="4"/>

  <module name="TreeWalker">
    <module name="Indentation" />
  </module>
</module>

E:\temp\CheckstyleReport>type Test.java
public class Test {
        private Test client;

        private Test(String string) {
        }

        private void test() {
                try {
                        client = new Test(
                                null // unexpected violation about indentation
                        );
                } catch (Exception e) {
                }
        }
}
E:\temp\CheckstyleReport>set RUN_LOCALE="-Duser.language=en -Duser.country=US"

E:\temp\CheckstyleReport>java %RUN_LOCALE% -jar checkstyle-13.0.0-all.jar -c config.xml Test.java
Starting audit...
[ERROR] E:\temp\CheckstyleReport\Test.java:10:17: 'new' child has incorrect indentation level 16, expected level should be 20. [Indentation]
Audit done.
Checkstyle ends with 1 errors.
```

---

**Describe what you expect in detail.**
The violation shouldn't be reported: the parameter passed to the constructor is already indented correctly.

NOTE: if we remove the try/catch block the violation disappears.
NOTE2: the file is formatted by tabs

---

--- a/src/test/java/com/puppycrawl/tools/checkstyle/checks/indentation/IndentationCheckTest.java
+++ b/src/test/java/com/puppycrawl/tools/checkstyle/checks/indentation/IndentationCheckTest.java
@@ -4256,6 +4256,20 @@ public class IndentationCheckTest extends AbstractModuleTestSupport {
         verifyWarns(checkConfig, fileName, expected);
     }
 
+    @Test
+    public void testTryBlockConstructorParameters() throws Exception {
+        final DefaultConfiguration checkConfig = createModuleConfig(IndentationCheck.class);
+        checkConfig.addProperty("basicOffset", "4");
+        checkConfig.addProperty("braceAdjustment", "0");
+        checkConfig.addProperty("caseIndent", "4");
+        checkConfig.addProperty("lineWrappingIndentation", "8");
+        checkConfig.addProperty("tabWidth", "4");
+
+        final String fileName = getPath("InputIndentationTryCtorParams.java");
+        final String[] expected = CommonUtil.EMPTY_STRING_ARRAY;
+        verifyWarns(checkConfig, fileName, expected);
+    }
+
     private static final class IndentAudit implements AuditListener {
 
         private final IndentComment[] comments;

--- a/src/test/resources/com/puppycrawl/tools/checkstyle/checks/indentation/indentation/InputIndentationTryCtorParams.java
+++ b/src/test/resources/com/puppycrawl/tools/checkstyle/checks/indentation/indentation/InputIndentationTryCtorParams.java
@@ -1,0 +1,29 @@
+/**                                                                          //indent:0 exp:0
+ * This test-input is intended to be checked using following configuration:  //indent:1 exp:1
+ *                                                                           //indent:1 exp:1
+ * basicOffset = 4                                                           //indent:1 exp:1
+ * braceAdjustment = 0                                                       //indent:1 exp:1
+ * caseIndent = 4                                                            //indent:1 exp:1
+ * lineWrappingIndentation = 8                                               //indent:1 exp:1
+ * tabWidth = 4                                                              //indent:1 exp:1
+ */                                                                          //indent:1 exp:1
+
+package com.puppycrawl.tools.checkstyle.checks.indentation.indentation;      //indent:0 exp:0
+
+public class InputIndentationTryCtorParams {                                 //indent:0 exp:0
+
+    private InputIndentationTryCtorParams client;                            //indent:4 exp:4
+
+    private InputIndentationTryCtorParams(String string) {                   //indent:4 exp:4
+    }                                                                        //indent:4 exp:4
+
+    private void test() {                                                    //indent:4 exp:4
+        try {                                                                //indent:8 exp:8
+            client = new InputIndentationTryCtorParams(                      //indent:12 exp:12
+                    null                                                     //indent:20 exp:20
+            );                                                               //indent:12 exp:12
+        }                                                                    //indent:8 exp:8
+        catch (Exception e) {                                                //indent:8 exp:8
+        }                                                                    //indent:8 exp:8
+    }                                                                        //indent:4 exp:4
+}                                                                            //indent:0 exp:0

